test/features/events/events.js

```diff
@@ -2,7 +2,7 @@
 
 const preq   = require('preq');
 const http   = require('http');
-const uuid   = require('cassandra-uuid').TimeUuid;
+const uuidTools   = require('../../../lib/uuidUtils');
```

Comment:
Consistency, please: `s/uuidTools/uuidUtils/`

Reply:
Changed the name